WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView.swift

```diff
@@ -71,13 +77,13 @@ private enum Localization {
         "payments onboarding.")
 
     static let cashOnDeliveryLearnMore = NSLocalizedString(
-        "%1$@ about adding Pay in Person to your checkout",
+        "%1$@ about accepting payments with your mobile device and ordering card readers.",
         comment: "A label prompting users to learn more about card readers. %1$@ is a placeholder that is " +
         "always replaced with \"Learn more\" string, which should be translated separately and considered " +
         "part of this sentence.")
 }
 
 private enum Constants {
-    static let imageHeight: CGFloat = 140.0
+    static let imageHeight: CGFloat = 88.0
```

Comment:
Since there's more text now and the image is smaller, we reduce this to fit the design better. I went ahead with 88 since is the original size on the figma file, and is a multiple of 44 which is the minimum touchable surface within the human interface guidelines.

Reply:
Thanks for the explanation @iamgabrielma I'm wondering, is there a way to make this dynamic so we don't have to change this depending on the image size? That way we can replace the image, without having to edit the constants

Reply:
That's a good question! The main problem I found with this is that the current view relies on another component ( `InPersonPaymentsOnboardingErrorMainContentView` ) to display the inner content, and this is used across 20+ other views related to IPP state. When this was created, all screens/images had similar characteristics design-wise so the component takes several assumptions about how they should look. 

Since this seems to be a one-off, I'll leave it as it is for now, but if we continue making changes to these views then it might be a good chance to revisit the component.